docs/_sources/lsh.rst.txt

```diff
@@ -123,3 +123,147 @@ bulk insertion.
 
 Note that querying the LSH object during an open insertion session may result in
 inconsistency.
+
+.. _minhash_lsh_async:
+
+Asynchronous MinHash LSH at scale
+---------------------------------
+.. note::
+    The module supports Python version >=3.6, and is currently experimental. So the interface may change slightly in the future.
+
```

Comment:
An overview on potential usage cases for the async module can be extremely helpful to users. For example, in what scenarios some one would want to use async rather than sync (e.g., high throughput insert?). An example usage case would be great! This generally increases adoption of the module, which is good for us.

Reply:
Add some extra docs